index.html

```diff
@@ -429,5 +429,36 @@ <h2 class="about-title">
 				</li>
 			</ul>
 		</section>
+
+		<section class="contact">
+			<div class="contact-container">
+				<div class="contact-header">
+					<h2>Contact me</h2>
+					<p>
+						If you have an application you are interested in developing, a
+						feature that you need built or a project that needs coding. I’d love
+						to help with it
+					</p>
+				</div>
+				<form action="https://formspree.io/f/xzbygpgy" method="POST">
+					<ul class="form-container">
+						<li class="form-element">
+							<input name="name_input" type="text" minlength="10" maxlength="30" placeholder="Name" required class="form-input" />
+						</li>
+						<li class="form-element">
+							<input name="email_input" type="email" minlength="10" maxlength="30" placeholder="Email" required class="form-input" />
+						</li>
```

Comment:
- [ ] **OPTIONAL** It's up to you, but don't you think someone may get a name less than 10 characters! For the emails, it doesn't make sense to put a minimum and maximum length since it has validation which is the email type.

> "Sam Paul" = 8 chars

It'd be way too better if you removed these validations from both the inputs.